cypress/integration/full/canvas_test_spec.js

```diff
@@ -48,8 +48,8 @@ context('Test Canvas', function(){
                 canvas.getFourUpViewToggle().should('be.visible');
                 canvas.getFourUpView().should('not.be.visible');
             });
-
-            it('verify share button', function(){
+            // TODO: Changes to the Share Button
+            it.skip('verify share button', function(){
```

Comment:
Try to fix the tests if you can. We shouldn't make Evangeline do all the work.

Reply:
Corrected the `Canvas` buttons so the tests run again!